src/html_support_files/odoc_search.js

```diff
@@ -18,23 +17,28 @@ function createWebWorker() {
   return worker;
 }
 
-var worker = createWebWorker();
+var worker;
+
+document.querySelector(".search-bar").addEventListener("focus", (ev) => {
+  if (typeof worker == "undefined") {
+    worker = createWebWorker();
+    worker.onmessage = (e) => {
+      let results = e.data;
+      let search_result = document.querySelector(".search-result-inner");
+      search_result.innerHTML = "";
+      let f = (entry) => {
+        /* entry */
+        let container = document.createElement("a");
```

Comment:
`search_result` is `search_results` (or `container`) and `container` is `search_result`.

Reply:
Indeed! Fixed in 0ef34cf.